### PR TITLE
Add restart logging to the slow /healthz conformance test

### PR DESCRIPTION
**Pull request by jayunit100**
sort of "fixes" #110839 ?... at least we'll be able to pinpoint the problem of waiting for restarts better, otherwise there is a black hole in the logs, and its hard to interpret the four minute progress freeze: 

```
STEP: checking the pod's current state and verifying that restartCount is present
Aug 16 01:03:08.695: INFO: Initial restart count of pod test-webserver-534db843-7314-4b25-a833-fff7ec0f9442 is 0
... (four minutes goes by....) 
Aug 16 01:07:09.983: INFO: Waiting up to 3m0s for all (but 0) nodes to be ready
STEP: Destroying namespace "container-probe-9632" for this suite.
```

### CODE REVIEW

File: test/e2e/common/node/container_probe.go [hunk 1]
```diff
@@ -971,6 +971,11 @@ func RunLivenessTest(f *framework.Framework, pod *v1.Pod, expectNumRestarts int,
 		pod, err = podClient.Get(context.TODO(), pod.Name, metav1.GetOptions{})
 		framework.ExpectNoError(err, fmt.Sprintf("getting pod %s", pod.Name))
 		restartCount := podutil.GetExistingContainerStatus(pod.Status.ContainerStatuses, containerName).RestartCount
+		// Since this is one of the slowest tests, its good to have logs between the 3 minute convergence time...
+		// see https://github.com/kubernetes/kubernetes/issues/110839
```
Comment (aojea) [inline test/e2e/common/node/container_probe.go:975]: @pohly didn't you fix the problem of slow test not showing output?

Comment (pohly) [reply test/e2e/common/node/container_probe.go:975]: Ginkgo should start showing where a test is stuck after 5 minutes (see [`--ginkgo.poll-progress-after`](https://github.com/kubernetes/kubernetes/blob/bb2101bac12748aa77adfa15b07b5b8d8dcd0672/hack/ginkgo-e2e.sh#L206).

This seems to be about node e2e, right? Does that run tests with `ginkgo-e2e.sh`? If not, then `--ginkgo.poll-progress-after` may have to be added to the e2e_node invocation.

File: test/e2e/common/node/container_probe.go [hunk 2]
```diff
@@ -971,6 +971,11 @@ func RunLivenessTest(f *framework.Framework, pod *v1.Pod, expectNumRestarts int,
...
+		if time.Since(start)*time.Second % 30 == 0 {
```
Comment (pohly) [inline test/e2e/common/node/container_probe.go:976]: Don't you have to divide by time.Second to get seconds?

File: test/e2e/common/node/container_probe.go (hunk 1 above)
Comment (aojea) [reply test/e2e/common/node/container_probe.go:975]: @jayunit100 are you running ginkgo directly instead of using the repo scripts?

**Comment by SergeyKanzhelev** [comment]
I don't understand how adding this log message is helping to troubleshoot the test. If this is to make test more user friendly, consider just adding the `framework.Logf` before the wait that will tell that the potentially long wait is about to begin. I think this falls under the [trivial edits](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits) policy

/close

**Pull request closed without merging**